dotty-docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:19.10
```

Comment:
20.04 is out and is a long term support release, would be good to use that if possible.